packages/api/tests/sample.test.js

```diff
@@ -0,0 +1,5 @@
+describe('Sample Test', () => {
+  it('should test that true === true', () => {
+    expect(true).toBe(true)
+  })
+})
```

Comment:
What do you think about just deleting this sample test?

Reply:
deleted